--- /Users/mutlupolatcan/.hermes/scripts/hermes-gateway-keychain.sh
+++ /Users/mutlupolatcan/.hermes/scripts/hermes-gateway-keychain.sh
@@ -11,4 +11,4 @@
 hermes_executable="/Users/mutlupolatcan/.local/bin/hermes"
 if [[ "$profile" == "general" ]]; then
-  hermes_executable="/Users/mutlupolatcan/.hermes/runtime/releases/hermes-agent-293daf98684321b628c29adbd3eef064b1d3fdc9/venv/bin/hermes"
+  hermes_executable="/Users/mutlupolatcan/.hermes/runtime/releases/hermes-agent-b208667b53acff437e38a339c7d70d618ce7f90e/venv/bin/hermes"
 fi
