--- a/hermes-gateway-keychain.sh
+++ b/hermes-gateway-keychain.sh
@@ -10,7 +10,7 @@
 
 hermes_executable="/Users/mutlupolatcan/.local/bin/hermes"
 if [[ "$profile" == "general" ]]; then
-  hermes_executable="/Users/mutlupolatcan/.hermes/runtime/releases/hermes-agent-8488dc3717b5b22ed1923b108dd6bd54c47adc30/venv/bin/hermes"
+  hermes_executable="/Users/mutlupolatcan/.hermes/runtime/releases/hermes-agent-11a67f98fd98de08affb7e948244b3bde2853906/venv/bin/hermes"
 fi
 
 service="com.polatcangames.hermes.op-service-account"
